gixy/plugins/proxy_pass_normalized.py

```diff
@@ -0,0 +1,44 @@
+import re
+import gixy
+from gixy.plugins.plugin import Plugin
+
+class ProxyPassDecodedRegex(Plugin):
```

Comment:
It should be `ProxyPassNormalized` to match with the filename

Reply:
It needs to be proxy_pass_normalized so the testsuite works, fixed.